packages/scaffold-config/src/dependencies.ts

```diff
@@ -97,6 +97,51 @@ export const WIZARD_DEPENDENCY_NEXT = {
   minVersion: '>=10.0.0',
 } as const
 
+export const WIZARD_DEPENDENCY_ANGULAR_CLI = {
+  type: 'angular',
+  name: 'Angular CLI',
+  package: '@angular/cli',
+  installer: '@angular/cli',
+  description: 'CLI tool that you use to initialize, develop, scaffold, and maintain Angular applications.',
+  minVersion: '>=13.0.0',
+} as const
```

Comment:
Any reason we need the const assertion here but not on the other items being added?

Reply:
Oversight on my part! Addressed in [be09724](https://github.com/cypress-io/cypress/pull/22897/commits/be09724cbce0eed112b16425c6acfb997cf4f7b3)